stopwords.go

```diff
@@ -62,6 +62,50 @@ func NewStopwords(path string) StopWords {
 }
 */
 
+// NewStopwords returns an instance of a stop words detector
+// new stopword lists can be added to the "resources/stopwords" directory as .txt with the filename
+// prefixed as "ISOLangCode_language_stopwords.txt"
+// ie. en_english_stopwords.txt
+func NewStopwords() StopWords {
+	cachedStopWords := make(map[string]*set.Set)
+	path := "resources/stopwords"
```

Comment:
[nit] maybe a `const`?

Reply:
yeah, moved `path` outside the function and made it a `const`